apps/web/src/hooks/useThemeChange.tsx

```diff
@@ -1,21 +1,16 @@
 import { ColorScheme, useMantineColorScheme } from '@mantine/core';
-import { IconOutlineDarkMode, IconOutlineLightMode, IconOutlineTonality } from '@novu/novui/icons';
+import { IconOutlineDarkMode, IconOutlineLightMode, IconOutlineTonality, IconType } from '@novu/novui/icons';
 import { ColorSchemePreferenceEnum, THEME_TITLE_LOOKUP, useLocalThemePreference } from '@novu/design-system';
 import { useSegment } from '../components/providers/SegmentProvider';
 import { useEffect, useMemo } from 'react';
 import { useDebounce } from './useDebounce';
 
 type ThemeChange = { colorScheme: ColorScheme; themeStatus: ColorSchemePreferenceEnum };
 
-const getThemeIcon = (themeStatus: ColorSchemePreferenceEnum) => {
-  if (themeStatus === ColorSchemePreferenceEnum.DARK) {
-    return <IconOutlineDarkMode />;
-  }
-  if (themeStatus === ColorSchemePreferenceEnum.LIGHT) {
-    return <IconOutlineLightMode />;
-  }
-
-  return <IconOutlineTonality />;
+export const ICON_BY_THEME_PREFERENCE: Record<ColorSchemePreferenceEnum, IconType> = {
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This allows for more flexibility with regard to sizing and styling when used ‏